cpp/include/gar/reader/arrow_chunk_reader.h

```diff
@@ -348,7 +359,10 @@ class AdjListOffsetArrowChunkReader {
       chunk_table_.reset();
     }
     if (chunk_index_ >= vertex_chunk_num_) {
-      return Status::KeyError();
+      return Status::KeyError("Internal vertex id ", id,
```

Comment:
I am not sure that why not to use  `OutofRange` here

Reply:
good point, as above mention, I think I confused the KeyError and OutOfRange here, in the `seek` scenario, the given parameter is in a range and it should raise a OutOfRange here. I will double check all the `seek` operation.